### PR TITLE
Feature/cc 7292 insights logging whitelist

### PR DESCRIPTION
**Pull request by dpetzold**
Adds whitelist support to the Insights logging

@sheats @brianz @gposton can you guys look at this.

Thanks,

### CODE REVIEW

File: tests/settings.py [hunk 1]
```diff
@@ -36,7 +36,12 @@
 }
 
 CACHEOPS_INSIGHTS_ENABLED = True
-CACHEOPS_INSIGHTS_BATCH_SIZE = 10
+CACHEOPS_INSIGHTS_BATCH_SIZE = 1
```
Comment (sheats) [inline tests/settings.py]: Seems if you are testing "batch" your default should be at least 2...

**Comment by sheats** [comment]
Cool, makes sense to me... :+1:

**Comment by dpetzold** [comment]
Closing in lieu of updates. Going start logging the create time.

**Comment by dpetzold** [comment]
@sheats @brianz @gposton this is ready again. If the model is whitelisted the ttl is fetched from redis so we can calculate the age of the object. Which will allow us to better adjust the cache settings. The related pull requests to ClearCare is below:

https://github.com/clearcare/clearcare/pull/1270

Please check! Thanks.

File: cacheops/analytics.py [hunk 2]
```diff
@@ -15,37 +18,41 @@ class InsightsReporter(object):
...
+        if self.enabled_for_model(model_name):
+            self.events.append({
+                'eventType': 'CacheEvent',
+                'action': action,
+                'app_name': model_name,
+                'cache_age': cache_age,
+                'cache_key': cache_key,
+                'timestamp': time.time(),
+            })
+            if len(self.events) == INSIGHTS_BATCH_SIZE:
```
Comment (brianz) [inline cacheops/analytics.py]: Better to change this to `>=` for safety.

**Comment by dpetzold** [comment]
@brianz its been updated

**Comment by brianz** [comment]
Looks good! :+1:

**Pull request merged**